### PR TITLE
Feature/lesson detail ui

### PR DESCRIPTION
**Pull request by delmontemica**
Created UI for lesson detail page

### CODE REVIEW

File: frontend_vue/src/assets/scss/style.scss [hunk 1]
```diff
@@ -52,8 +52,46 @@ div{
...
     font-size: 24px;
     font-weight: 500;
   }
+  &-subtitle{
+    margin-left: 5px;
+  }
+  &-boxes{
+    height: 150px;
+    width: 170px;
+    margin: 10px 10px 10px 10px;
```
Comment (hugh-n) [inline frontend_vue/src/assets/scss/style.scss]: You can change this to margin: 10px;

File: frontend_vue/src/components/LessonDetail.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                  <v-icon medium color="black">{{ test.icon }}</v-icon>
+                  <h3> {{ test.title }}</h3>
+                </div>
+              </v-card-title>
+            </v-card>
+        </v-layout>
+        <v-divider class="lessons-divider"></v-divider>
+        <v-flex xs12 mt-4 ml-3>
+          <div class="lessons-subtitle">
+            <span>Terms in the set (20)</span>
```
Comment (hugh-n) [inline frontend_vue/src/components/LessonDetail.vue]: Just want to ask why is this currently static? (20)

File: frontend_vue/src/components/LessonDetail.vue [hunk 3]
```diff
@@ -0,0 +1,67 @@
+<template>
+    <v-content>
+        <v-layout row wrap>
+            <v-flex xs12 mt-4 ml-3>
+                <span class="lessons-title">レッスン01</span>
+                <div class="lessons-subtitle">
+                  <span>20 terms</span> | <span><a href="">JELLYFISH</a></span>
```
Comment (hugh-n) [inline frontend_vue/src/components/LessonDetail.vue]: Just want to ask why term count is currently static? (20)

File: frontend_vue/src/assets/scss/style.scss (hunk 1 above)
Comment (delmontemica) [reply frontend_vue/src/assets/scss/style.scss]: Fixed.

File: frontend_vue/src/components/LessonDetail.vue (hunk 2 above)
Comment (delmontemica) [reply frontend_vue/src/components/LessonDetail.vue]: This is fixed.
The fix will be on the next PR.

File: frontend_vue/src/components/LessonDetail.vue (hunk 3 above)
Comment (delmontemica) [reply frontend_vue/src/components/LessonDetail.vue]: Fixed.

**Pull request merged**